packages/mobile/src/navigator/types.tsx

```diff
@@ -161,10 +161,10 @@ export type StackParamList = {
   }
   [Screens.PincodeEnter]: {
     withVerification?: boolean
-    onSuccess: (pin: string) => void
-    onCancel: () => void
+    onSuccess?: (pin: string) => void
+    onCancel?: () => void
   }
-  [Screens.PincodeSet]: { isVerifying: boolean } | undefined
```

Comment:
Did you remove the `| undefined` on purpose?

Reply:
unintentional

Reply:
will add it back! was trying to look for where it is used though, any ideas? :o